CLI_COMMANDS.md

```diff
@@ -0,0 +1,340 @@
+
```

Comment:
@thefoxis `CLI_COMMANDS` shows all commands and sub-commands, inclusive of their descriptions, options and defaults. If you can think of any improvements from this first cut I'd be happy to include them! On first look it seems like a good start 🥲

Reply:
@benschwarz I think you can push it as is for now. we can improve later if we want to. 👍🏻

Reply:
Aight then! Thanks for your help. I'll update the changelog, cleanup and get this release shipped 🛥️